.github/workflows/ci.yml

```diff
@@ -26,13 +27,16 @@ jobs:
       - name: Install dependencies
         run: |
             python -m pip install --upgrade pip
-            if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
@davidfnatixis removed uneeded checks for `requirements.txt` because we own the code and we now if it exists or not.